src/utils/api-utils.js

```diff
@@ -0,0 +1,12 @@
+import axios from "axios";
+
+export default function FetchAllArticles (){
+    return axios
+    .get("https://newsflash-e6p1.onrender.com/api/articles")
+    .then((response) => {
+      return response.data.articles
+    })
+    .catch((error) => {
+      console.log(error);
+    });
```

Comment:
There's a later ticket when you deal with errors, if you move the catch block into the useEffect then you could use that error inside the component to alter some state. Might make more sense later!

Reply:
moved and will keep in mind